src/components/PostsList.tsx

```diff
@@ -1,85 +1,68 @@
-import React from 'react';
+import classNames from 'classnames';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[],
+  selectedPost: Post | null,
+  setSelectedPost: (value: Post | null) => void,
+  setIsCommentButtonClicked: (value: boolean) => void,
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  posts,
+  selectedPost,
+  setSelectedPost,
+  setIsCommentButtonClicked,
+}) => {
+  const selectedPostHandler = (post: Post) => {
```

Comment:
create this function in the App.tsx and pass as prop instead of setSelectedPost, and setIsCommentButtonClicked,